airflow/providers/databricks/hooks/databricks.py

```diff
@@ -305,6 +311,11 @@ def _do_api_call(self, endpoint_info, json):
         """
         method, endpoint = endpoint_info
 
+        if 'host' in self.databricks_conn.extra_dejson:
```

Comment:
`self.host` might want  to be a cached prop also.  or not be an attribute at all.  it feels a  little uncomfortable to mutate attributes like this, i.e. as side effects of a method

Reply:
Ah you're right. Copy pasta. This doesn't need to be an instance attribute. `self.host` is only referenced in `_do_api_call()`.